docs/k0s-multi-node.md

```diff
@@ -31,15 +32,15 @@ k0s token create --role=worker
 
 This will output a long [token](#tokens) which we will use to join the worker. To enhance security, we can also set an expiration time on the tokens by using:
```

Comment:
```suggestion
This will output a long [token](#tokens) which we will use to join a worker to the cluster. To enhance security, we can also set an expiration time for the token by using:
```